apps/website/src/pages/index.tsx

```diff
@@ -0,0 +1,435 @@
+      className="hover:text-alveus-green transition-colors"
+    >
+      <Image
+        src={src}
+        alt={alt}
+        draggable={false}
+        className="w-full h-auto aspect-square object-cover max-w-[10rem] mx-auto rounded-xl"
+      />
+      <Heading level={4} className="text-center text-xl">{title}</Heading>
+      <p className="text-center">{description}</p>
+    </Link>)
+  }), {});
+
+const merch = Object.entries({
+  hoodie: {
+    src: hoodieMerchImage,
+    alt: "Unisex Essential Echo Hoodie (White)",
+  },
+  case: {
+    src: caseMerchImage,
+    alt: "Biodegradable iPhone Case (iPhone 11 Pro Max)",
+  },
+  tshirt: {
+    src: tshirtMerchImage,
+    alt: "Organic Cotton T-Shirt Dress (Black)",
+  },
+  croptop: {
+    src: croptopMerchImage,
+    alt: "Organic Crop Top (Black)",
+  },
+  tote: {
+    src: toteMerchImage,
+    alt: "Large Eco Tote (Oyster)",
+  },
+  beanie: {
+    src: beanieMerchImage,
+    alt: "Organic Ribbed Beanie (Black)",
+  },
+})
+  .reduce((obj, [ key, { src, alt } ]) => ({
+    ...obj,
+    [key]: (<Image
+      src={src}
+      alt={alt}
+      draggable={false}
+      className="w-full h-auto max-w-[10rem] mx-auto"
+    />)
+  }), {});
+
+const help = {
+  donate: {
+    icon: IconPayPal,
+    title: "Donate via PayPal",
+    link: "/paypal",
+    external: true,
+  },
+  amazon: {
+    icon: IconAmazon,
+    title: "Amazon Wishlist",
+    link: "/wishlist",
+    external: true,
+  },
+  contact: {
+    icon: IconEmail,
+    title: "Offer Help",
+    link: "https://www.alveussanctuary.org/contact-us/",
+    external: false,
+  },
+};
+
+const Home: NextPage = () => {
+  const twitchEmbedId = useId();
+  const twitchEmbedRef = useRef<HTMLDivElement>(null);
+  const [ twitchEmbedLoaded, setTwitchEmbedLoaded ] = useState(false);
+  const [ twitchEmbedPlaying, setTwitchEmbedPlaying ] = useState(false);
+  useEffect(() => {
+    if (!twitchEmbedLoaded || typeof Twitch === "undefined") {
+      if (typeof Twitch !== "undefined") setTwitchEmbedLoaded(true);
+      return;
+    }
+    if (!twitchEmbedRef.current) return;
+
+    // Reset the embed (fixes dev hot reload)
+    twitchEmbedRef.current.innerHTML = "";
+
+    // Load the embed from Twitch
+    // https://dev.twitch.tv/docs/embed/everything
+    const embed = new Twitch.Embed(`twitch-embed-${twitchEmbedId}`, {
+      width: "100%",
+      height: "100%",
+      channel: "alveussanctuary",
+      layout: "video",
+      allowfullscreen: false,
+      autoplay: true,
+      muted: true,
+    });
+
+    // Track when playing, as we disable pointer events on the embed
+    embed.addEventListener(Twitch.Embed.VIDEO_PLAY, () => {
+      const player = embed.getPlayer();
+      console.log("The video is playing", player);
+      setTwitchEmbedPlaying(true);
+    });
+  }, [twitchEmbedId, twitchEmbedLoaded])
+
+  return (
+    <>
+      <Head>
+        <title>Alveus.gg</title>
+        <meta name="robots" content="noindex" />
```

Comment:
Should we set`noindex` on all pages (or in a robots.txt) and can we remove it from the pages then?

Reply:
I've been setting it on the pages that I'm porting for now to avoid risking any SEO collision. I think it's fine to allow the "normal" pages on here (such as /live) to be indexed if found, as they don't collide with the main site.

Reply:
Yeah makes sense. I was unsure about the giveaways etc. But they will be disabled soon anyways, depending on how long donators take to enter their addresses. I think Google wont find the site by itself until then and I would hold off registering it in the web search console.